Proof reading II (#733)
ansible/ansible-documentation: docs/docsite/rst/os_guide/index.rst

    Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. We ask that you open an issue or pull request if you come upon a term that we have missed. For more details, see `our CTO Chris Wright's message <https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language>`_.

Welcome to the Ansible guide for Microsoft Windows and BSD.

> Because Windows is not a POSIX-compliant operating system, Ansible interacts with Windows hosts differently as compared to Linux/Unix hosts.

Reviewer: ```suggestion
Because Windows is not a POSIX-compliant operating system, Ansible interacts with Windows hosts differently than Linux/Unix hosts.
```

> Likewise managing hosts that run BSD is different from managing other Unix-like host operating systems.

Reviewer: I believe this should be `than` not `from` and a comma is missing.

```suggestion
Likewise, managing hosts that run BSD is different than managing other Unix-like host operating systems.
```
